yarn.lock

```diff
@@ -3897,6 +3897,14 @@ http-proxy-agent@^4.0.1:
     agent-base "6"
     debug "4"
 
+https-proxy-agent@5.0.1:
+  version "5.0.1"
+  resolved "https://packages.atlassian.com/api/npm/npm-remote/https-proxy-agent/-/https-proxy-agent-5.0.1.tgz#c59ef224a04fe8b754f3db0063a25ea30d0005d6"
```

Comment:
this is wrong... it should be pointing to npmjs.org not PAC for this library

Reply:
Someday, I'll learn not to stumble over this stupid file....